### DISCUSSION TITLE
General discussion

### DISCUSSION THREAD

**Discussion by ehsan6sha** [functionland/fx-fotos#46] [General]
This discussion page is for general discussions about the project.
If you are not sure how you can contribute, you can just introduce yourself and your area of interests to get connected :)

**Comment by pegah-m-sadeghi** [comment]
I am interested to contribute here. Any issue you recommend?

**Reply by ehsan6sha** [reply]
Kudos. I mentioned you on an issue which could be a good start

**Comment by emadbaqeri** [comment]
Hey every one, any idea about wrting good wiki for the project? I think I can help with that too, but first I should have the mind map.

**Reply by ehsan6sha** [reply]
Wow!!! That would be amazing Emad if you could! Even if you can start a draft we can then get together and improve it. I really appreciate your help in this.

**Reply by emadbaqeri** [reply]
I think that would be a bit challengin for me but I can work on this maybe on weekend. But I can do this and I can accept the responsibility of this feature too.
